tools/importers/SynchronizeTypographyTokens.spec.ts

```diff
@@ -0,0 +1,115 @@
+import {
+	ImportTypographyVariables,
+	ImportTypographyTokens,
+	requestForBin,
+} from './SynchronizeTypographyTokens';
+const fs = require('fs');
+const tokensFilesConfig = require('./configs/FakeSynchronizeTypographyTokensConfig.json');
+
+describe('Typography Tokens', () => {
+	it('check config & create files', async () => {
+		let requestResponse = await requestForBin(
+			tokensFilesConfig.bin,
+			tokensFilesConfig.jsonBinApiUrl,
+		);
+
+		ImportTypographyVariables(
+			tokensFilesConfig,
+			requestResponse.data.record.values.LMSDesignSystemTypography,
+		);
+
+		ImportTypographyTokens(
+			tokensFilesConfig,
+			requestResponse.data.record.values.LMSDesignSystemTypography,
+		);
+	});
+	it('check variable SCSS file', async () => {
+		await new Promise((r) => setTimeout(r, 500));
```

Comment:
sleep in tests code? We should avoid such patterns. What if operation takes longer than `500ms`?

Reply:
I don't know where is the problem, without this timeout tests sometimes passed sometimes not. They are launched sequentially but maybe when file is written on the disc in the first test, in the second is not available yet. But if I will change request to the static file, this problem will not occurs.

Reply:
https://github.com/bethinkpl/design-system/pull/222#discussion_r1024808141